src/backend/crew_ai/library_context/dynamic_context.py

```
@@ -81,8 +126,11 @@ def get_library_documentation(self) -> Dict:
             return doc_data
             
         except ImportError as e:
-            logger.error(f"Library {self.library_name} is not installed: {e}")
-            raise ImportError(f"Library {self.library_name} not found. Please install it first.")
+            logger.error(f"Library {self.library_name} is not installed and no pre-generated libdoc found: {e}")
```

Ruff (0.15.7)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)